modules/administration/pages/actions.adoc

```diff
@@ -117,9 +120,47 @@ Wildcards can be used to run commands across any number of systems.
 Always take extra care to check your commands before issuing them.
 ====
 
+Before you begin, ensure your client is subscribed to the appropriate tools child channel for its installed operating system.
+For more information about subscribing to software channels, see xref:client-configuration:channels.adoc[].
+
+
+// This seems really hacky. Is it still correct? --LKB 2020-07-28
+.Procedure: Configuring Clients to Accept Remote Commands
+. On the client, at the command prompt, use the package manager to install the [systemitem]``rhncfg``, [systemitem]``rhncfg-client``, and [systemitem]``rhncfg-actions`` packages, if not already installed.
+For example:
++
+----
+zypper in rhncfg rhncfg-client rhncfg-actions
+----
+. On the client, at the command prompt, as root, create a path in the local configuration directory:
++
+----
+mkdir -p /etc/sysconfig/rhn/allowed-actions/script
+----
++
+. Create an empty file called [path]``run`` in the new directory.
+This file grants the {productname} Server permission to run remote commands:
++
+----
+touch /etc/sysconfig/rhn/allowed-actions/script/run
+----
```

Comment:
This is technically what happens when you call:

$> mgr-actions-control --enable-run

Reply:
so should we document that instead?

Reply:
Doesn't this happen automatically if you enable remote command with the bootstrap script.  Of course, it would not hurt if we document in the Ref Guide what happens behind the scenes.  But I think we shouldnot advise the user to run those setup steps manually.
This is just my opinion.  Let's wait what mc will tell us :)

Reply:
Yes, this should happen when you use the bootstrap script and have it configured. But I think it is a good idea to document the packages and the command as well. Just in case somebody wants to enable it later.

Please add also that this is only required for traditional clients. Nothing of it is needed when you use a salt client.